tmt/steps/prepare/install.py

```diff
@@ -26,44 +46,140 @@ def options(cls, how=None):
         """ Prepare command line options """
         return [
             click.option(
-                '-p', '--package', metavar='NAME', multiple=True,
-                help='Package name to be installed.'),
+                '-p', '--package', metavar='PACKAGE', multiple=True,
+                help='Package name or path to rpm to be installed.'),
+            click.option(
+                '-D', '--directory', metavar='PATH', multiple=True,
+                help='Path to a local directory with rpm packages.'),
             click.option(
                 '-c', '--copr', metavar='REPO', multiple=True,
                 help='Copr repository to be enabled.')
             ] + super().options(how)
 
     def show(self):
         """ Show provided scripts """
-        super().show(['package', 'copr'])
+        super().show(['package', 'directory', 'copr'])
 
     def wake(self, data=None):
         """ Override options and wake up the guest """
-        super().wake(['package', 'copr'])
+        super().wake(['package', 'directory', 'copr'])
 
         # Convert to list if necessary
-        tmt.utils.listify(self.data, split=True, keys=['package', 'copr'])
+        tmt.utils.listify(
+            self.data, split=True, keys=['package', 'directory', 'copr'])
+
+    def enable_copr_epel6(self, copr, guest):
+        """ Manually enable copr repositories for epel6 """
+        # Parse the copr repo name
+        matched = re.match("^(@)?([^/]+)/([^/]+)$", copr)
```

Comment:
seems like this would be nice as a general check @psss?

Reply:
Invalid repos will be nicely handled directly in copr plugin. I don't think we need to add an extra check there.